packs/pfs-season-3-bestiary/advanced-eltha-embercall-9-10-eltha-embercall-11-12.json

```diff
@@ -1758,7 +1758,7 @@
                             }
                         },
                         "description": {
-                            "value": "<p>You channel vitality energy to heal the living or damage the undead. If the target is a willing living creature, you restore 1d8 Hit Points. If the target is undead, you deal that amount of vitality damage to it, and it gets a basic Fortitude save. The number of actions you spend when Casting this Spell determines its targets, range, area, and other parameters.</p>\n<p><span class=\"action-glyph\">1</span> (somatic) The spell has a range of touch.</p>\n<p><span class=\"action-glyph\">2</span> (verbal, somatic) The spell has a range of 30 feet. If you're healing a living creature, increase the Hit Points restored by 8. [[/r (@item.level)d8[vitality]+@item.level*8[healing]]]{2-Action Healing}</p>\n<p><span class=\"action-glyph\">3</span> (material, somatic, verbal) You disperse vitality energy in a 30-foot emanation. This targets all living and undead creatures in the burst.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> The amount of healing or damage increases by 1d8, and the extra healing for the 2-action version increases by 8.</p>"
+                            "value": "<p>You channel vitality energy to heal the living or damage the undead. If the target is a willing living creature, you restore 1d8 Hit Points. If the target is undead, you deal that amount of vitality damage to it, and it gets a basic Fortitude save. The number of actions you spend when Casting this Spell determines its targets, range, area, and other parameters.</p>\n<p><span class=\"action-glyph\">1</span> (somatic) The spell has a range of touch.</p>\n<p><span class=\"action-glyph\">2</span> (verbal, somatic) The spell has a range of 30 feet. If you're healing a living creature, increase the Hit Points restored by 8. @Damage[(@item.level)d8[vitality]+@item.level*8[healing]]{2-Action Healing}</p>\n<p><span class=\"action-glyph\">3</span> (material, somatic, verbal) You disperse vitality energy in a 30-foot emanation. This targets all living and undead creatures in the burst.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> The amount of healing or damage increases by 1d8, and the extra healing for the 2-action version increases by 8.</p>"
```

Comment:
I don't think this works correctly. It probably needs to be rebuilt with a fresh copy of the spell.